main/src/main/java/org/mobilitydata/gtfsvalidator/validator/LocationHasStopTimesValidator.java

```diff
@@ -48,19 +46,41 @@ public class LocationHasStopTimesValidator extends FileValidator {
 
   private final GtfsStopTimeTableContainer stopTimeTable;
 
+  private final GtfsLocationGroupStopsTableContainer locationGroupStopTable;
+
   @Inject
   LocationHasStopTimesValidator(
-      GtfsStopTableContainer stopTable, GtfsStopTimeTableContainer stopTimeTable) {
+      GtfsStopTableContainer stopTable,
+      GtfsStopTimeTableContainer stopTimeTable,
+      GtfsLocationGroupStopsTableContainer locationGroupStopTable) {
     this.stopTable = stopTable;
     this.stopTimeTable = stopTimeTable;
+    this.locationGroupStopTable = locationGroupStopTable;
   }
 
   @Override
   public void validate(NoticeContainer noticeContainer) {
+    Set<String> stopIdsInStopTimesandLocationGroupStops = new HashSet<>();
+    Set<String> locationGroupIdsInStopTimes = new HashSet<>();
+
+    for (GtfsStopTime stopTime : stopTimeTable.getEntities()) {
+      stopIdsInStopTimesandLocationGroupStops.add(stopTime.stopId());
+      locationGroupIdsInStopTimes.add(stopTime.locationGroupId());
```

Comment:
We should not add null/empty IDs to the sets.

Reply:
added null checks